controllers/endpoint/endpoints_controller_test.go

```diff
@@ -262,11 +263,11 @@ func testPodTurningCLBFAfterWatchDuration(ctx context.Context, _ context.CancelF
 
 func testNoCLBFPodDeletionWhenEndpointNotReady(ctx context.Context, _ context.CancelFunc, g *WithT, reconciler *Reconciler, namespace string) {
 	createEp(ctx, g, reconciler, namespace, false)
-	ep := &v1.Endpoints{}
+	ep := &discoveryv1.EndpointSlice{}
 	g.Expect(reconciler.Client.Get(ctx, types.NamespacedName{Name: epName, Namespace: namespace}, ep)).To(Succeed())
 	turnEndpointToNotReady(ctx, g, reconciler.Client, ep)
 
-	el := v1.EndpointsList{}
+	el := discoveryv1.EndpointSliceList{}
```

Comment:
In function `deleteAllEp` you still have a reference to the now deprecated `v1.EndpointsList`. Can you also correct that?